docs/docs/community/best-practices.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 5
+title: Best Practices
+---
+
+# Best Practices for Pull Requests and Contributions
+
+This guide provides essential best practices for submitting Pull Requests (PRs) and contributing effectively to the project. By adhering to these practices, contributors ensure a more efficient workflow, maintain code quality, and facilitate smooth collaboration within the community.
+
+## Pre-Review with AI
+
+Before submitting a Pull Request (PR), we strongly recommend conducting a pre-review using an AI tool, such as [Coderabbit](https://www.coderabbit.ai/) or [Sweep](https://www.coderabbit.ai/). This preliminary step helps identify potential issues and provides recommendations for improvement before human intervention. Addressing AI-generated feedback allows contributors to enhance the quality of their submission, ensuring that the subsequent human review is more focused, efficient, and substantive.
+
+## Checklist for Each New PR
```

Comment:
Can we also add suggestions to use other AI tools like cursor, and LLMs like OpenAI, Claude, etc.